src/middlewares/accommodation.middleware.js

```diff
@@ -0,0 +1,17 @@
+import AccommodationService from '../services/accommodation.service';
+import ResponseService from '../services/response.service';
+
+const accommodationMiddleware = {
+  checkBookingExist: async (req, res, next) => {
+    const findBooking = await AccommodationService.findBookingByProperty({
+      accommodationId: req.params.accommodationId, userId: req.userData.id
+    });
+    if (!findBooking) {
+      ResponseService.setError(404, `Booking for userId=${req.userData.id} on accommodationId=${req.params.accommodationId} not found`);
```

Comment:
This error message is more technical for the user. Because these ids will not be visible to user from front end, so this message will confuse him. Try to make it simple and clear

Reply:
yes @gadishimwe let me change it

Reply:
@gadishimwe i have implemented the feedback you gave. you can have a look on it.